src/styles/main.scss

```diff
@@ -0,0 +1,30 @@
+.container {
+  --column-count: 1;
+
+  min-width: 1024px;
+  max-width: 1200px;
+  padding: 50px 40px 72px;
+
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  gap: 46px 48px;
```

Comment:
The gap values should also follow the 4px/8px scale unless there is a specific design requirement that dictates these values. Adjust them if necessary to maintain consistency with the code style rules.